litgpt/pretrain.py

```diff
@@ -342,6 +342,13 @@ def fit(
         if train.save_interval is not None and not is_accumulating and state["step_count"] % train.save_interval == 0:
             save_checkpoint(fabric, state, tokenizer_dir, out_dir / f"step-{state['step_count']:08d}" / "lit_model.pth")
 
+        # Final validation
+        if eval.final_validation:
```

Comment:
This can't be here, it is misplaced. 

As it is right now, this will evaluate ever iteration.

Reply:
Thanks, I'll take care of it!

Reply:
I think it was an indentation error, I think this should be fine now (ran it on a small model just to make sure)